From ffa3ed225452998f8de5cdde235f3ba0a6400d83 Mon Sep 17 00:00:00 2001
From: Yuki Shiino <yukishiino@chromium.org>
Date: Tue, 28 Jun 2022 05:11:12 +0000
Subject: [PATCH 45/59] sheriff: Disable QuarantineMacTest.*

(cherry picked from commit b9fd9cf1ba3fb180e4c94dc98016066f4f876fa5)

Bug: 1334495
Change-Id: I300191c2cdf9303e71e680846e0b6568fe0cd6c0
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3692655
Owners-Override: Yuki Shiino <yukishiino@chromium.org>
Commit-Queue: Yuki Shiino <yukishiino@chromium.org>
Reviewed-by: Takuto Ikuta <tikuta@chromium.org>
Cr-Original-Commit-Position: refs/heads/main@{#1011837}
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3708693
Commit-Queue: Takuto Ikuta <tikuta@chromium.org>
Owners-Override: Takuto Ikuta <tikuta@chromium.org>
Auto-Submit: Steven Holte <holte@chromium.org>
Cr-Commit-Position: refs/branch-heads/5005@{#1204}
Cr-Branched-From: 5b4d9450fee01f821b6400e947b3839727643a71-refs/heads/main@{#992738}
---
 .../quarantine/quarantine_mac_unittest.mm         | 15 ++++++++++-----
 1 file changed, 10 insertions(+), 5 deletions(-)

diff --git a/components/services/quarantine/quarantine_mac_unittest.mm b/components/services/quarantine/quarantine_mac_unittest.mm
index c7358a3ddfff..36572f572f6a 100644
--- a/components/services/quarantine/quarantine_mac_unittest.mm
+++ b/components/services/quarantine/quarantine_mac_unittest.mm
@@ -68,7 +68,8 @@ class QuarantineMacTest : public testing::Test {
   base::scoped_nsobject<NSURL> file_url_;
 };
 
-TEST_F(QuarantineMacTest, CheckMetadataSetCorrectly) {
+// TODO(crbug.com/1334495): Enable this test.
+TEST_F(QuarantineMacTest, DISABLED_CheckMetadataSetCorrectly) {
   QuarantineFile(
       test_file_, source_url_, referrer_url_, "",
       base::BindOnce(&CheckQuarantineResult, QuarantineFileResult::OK));
@@ -76,7 +77,8 @@ TEST_F(QuarantineMacTest, CheckMetadataSetCorrectly) {
   EXPECT_TRUE(IsFileQuarantined(test_file_, source_url_, referrer_url_));
 }
 
-TEST_F(QuarantineMacTest, SetMetadataMultipleTimes) {
+// TODO(crbug.com/1334495): Enable this test.
+TEST_F(QuarantineMacTest, DISABLED_SetMetadataMultipleTimes) {
   GURL dummy_url("http://www.dummy.example.com");
   QuarantineFile(
       test_file_, source_url_, referrer_url_, "",
@@ -97,7 +99,8 @@ TEST_F(QuarantineMacTest, IsFileQuarantined_NoAnnotationsOnFile) {
   EXPECT_FALSE(IsFileQuarantined(test_file_, GURL(), GURL()));
 }
 
-TEST_F(QuarantineMacTest, IsFileQuarantined_SourceUrlOnly) {
+// TODO(crbug.com/1334495): Enable this test.
+TEST_F(QuarantineMacTest, DISABLED_IsFileQuarantined_SourceUrlOnly) {
   QuarantineFile(
       test_file_, source_url_, GURL(), std::string(),
       base::BindOnce(&CheckQuarantineResult, QuarantineFileResult::OK));
@@ -108,7 +111,8 @@ TEST_F(QuarantineMacTest, IsFileQuarantined_SourceUrlOnly) {
   EXPECT_FALSE(IsFileQuarantined(test_file_, referrer_url_, GURL()));
 }
 
-TEST_F(QuarantineMacTest, IsFileQuarantined_FullMetadata) {
+// TODO(crbug.com/1334495): Enable this test.
+TEST_F(QuarantineMacTest, DISABLED_IsFileQuarantined_FullMetadata) {
   QuarantineFile(
       test_file_, source_url_, referrer_url_, std::string(),
       base::BindOnce(&CheckQuarantineResult, QuarantineFileResult::OK));
@@ -121,7 +125,8 @@ TEST_F(QuarantineMacTest, IsFileQuarantined_FullMetadata) {
   EXPECT_FALSE(IsFileQuarantined(test_file_, referrer_url_, referrer_url_));
 }
 
-TEST_F(QuarantineMacTest, IsFileQuarantined_Sanitize) {
+// TODO(crbug.com/1334495): Enable this test.
+TEST_F(QuarantineMacTest, DISABLED_IsFileQuarantined_Sanitize) {
   GURL host_url{"https://user:pass@example.com/foo/bar?x#y"};
   GURL host_url_clean{"https://example.com/foo/bar?x#y"};
   GURL referrer_url{"https://user:pass@example.com/foo/index?x#y"};
-- 
2.37.0

